### PR TITLE
Fix IOkitTest and RequestTest failures on windows

### PR DESCRIPTION
**Pull request by Frederick-S**
We can also setup ci for windows.

### CODE REVIEW

**Comment by hellokaton** [comment]
Thanks for your PR. I'll check it later

File: src/main/java/com/blade/mvc/http/HttpRequest.java [hunk 1]
```diff
@@ -51,7 +51,7 @@
 
     private Map<String, String>       headers    = null;
     private Map<String, Object>       attributes = null;
-    private Map<String, List<String>> parameters = null;
+    private Map<String, List<String>> parameters = new HashMap<>();
```
Comment (hellokaton) [inline src/main/java/com/blade/mvc/http/HttpRequest.java:54]: This line of code has been initialized in the init method in order to make the performance better

Comment (Frederick-S) [reply src/main/java/com/blade/mvc/http/HttpRequest.java:54]: I see. What's your opinion about fixing the null pointer exception in MethodArgumentTest and RequestInvokerTest? By the way, `mvn test` actually fails but ci shows passing.

Comment (hellokaton) [reply src/main/java/com/blade/mvc/http/HttpRequest.java:54]: I merged your PR, passed, thank you for your contribution.

**Pull request merged**